### PR TITLE
Updated the JIT provisioning description

### PR DESCRIPTION
**Pull request by shashankpradhan**
After discussions with a couple of MSPs along with Gobstopper, added a more intuitive explanation of the feature.

### CODE REVIEW

File: content/guides/saml.md [hunk 1]
```diff
@@ -78,12 +78,11 @@ For more information about configuring specific IdP's, refer to the following Kn
...
-### Just in Time Provisioning (JIT Provisioning)
-Normally users must be invited to Datadog, even for organizations with SAML enabled. 
-If a user that has not been invited to Datadog logs in via an Org's IdP, the SAML assertion will be validated, but they will be redirected to a SAML Error page.
+### Just-in-Time (JIT) Provisioning 
 
-Some organizations might not want to have to invite all of their users to Datadog. If you would like to make changes to how SAML works for your account, contact support.
-It is up to the organization to configure their IdP to not send assertions to Datadog if they don't want a particular user to access Datadog.
+With Just-in-Time provisioning, a user will be created within Datadog on the fly the first time they try to log in. This eliminates the need for admins to manually create user accounts one at a time.
+
+Some organizations might not want to invite all of their users to Datadog. If you would like to make changes to how SAML works for your account, please contact support. It is up to the organization to configure their IdP to not send assertions to Datadog if they don't want a particular user to access Datadog.
```
Comment (irabinovitch) [inline content/guides/saml.md]: can we link "contact" to the help page?

Comment (irabinovitch) [reply content/guides/saml.md]: @shashankpradhan ping?

Comment (technovangelist) [reply content/guides/saml.md]: i think he meant https://docs.datadoghq.com/help/

Comment (shashankpradhan) [reply content/guides/saml.md]: Done

**Pull request merged**